cdap-ui/app/cdap/components/PipelineConfigurations/PipelineConfigurations.scss

```diff
@@ -0,0 +1,459 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+@import "../../styles/variables.scss";
+
+$text-color: $grey-01;
+$border-color: $grey-05;
+$active-bg-color: $grey-07;
+$input-box-focus-color: #66afe9;
+$input-label-color: $grey-03;
+$action-button-color: $grey-02;
+$modeless-bg-color: $grey-06;
+$modeless-header-bg-color: $grey-08;
+$modeless-width: 830px;
+$modeless-height: 496px;
+$side-panel-width: 170px;
+$regular-font: 14px;
+$secondary-btn-color: $brand-primary;
+
+body.theme-cdap {
+  &.state-hydrator {
+    .pipeline-configurations-content {
+      background: $modeless-bg-color;
+      position: fixed;
```

Comment:
Should we use `position: absolute` here? Fixed just removes it from the entire DOM flow. We should use `absolute` positioning and have a fixed `right` or `left` position so that the modeless remains center to the button. 

Right now this is how it looks on a larger screen.

![screen shot 2018-02-14 at 11 52 18 am](https://user-images.githubusercontent.com/1452845/36225333-321f45ee-117f-11e8-84b3-900637806ff2.png)